Project-Service-implementation-Script/main.py

```diff
@@ -124,6 +126,10 @@ def programCreation(accessToken,parentFolder,externalId,pName,pDescription,roles
     programCreationurl = config.get(environment, 'elevateprojecthost') + config.get(environment, 'programCreationurl')
     messageArr.append("Program Creation URL : " + programCreationurl)
     # program creation payload
+
+    entities = districtentitiesPGM.split(',')
```

Comment:
@MallanagoudaB what is entities, can you add comment which explain what kind of value entities hold?